docs/usage/docker.md

```diff
@@ -138,6 +138,26 @@ To
 FROM ubuntu:focal
 ```
 
+### Debian codenames
+
+Renovate understands [Debian release code names and rolling updates schedule](https://wiki.debian.org/DebianReleases) and will offer upgrades to the latest stable release (e.g. from `debian:stretch` to `debian:bullseye`).
+
+For this to work you must follow this naming scheme:
+
+- The codename is in lowercase
```

Comment:
If all you need to do is to have the codename in lowercase, then we don't need the list format:

```suggestion
For this to work the codename must be in lowercase.
```